galaxy_importer/schema.py

```diff
@@ -189,14 +202,16 @@ def _check_dependencies_format(self, attribute, dependencies):
 
     @tags.validator
     def _check_tags(self, attribute, value):
-        """Check max tags and check against tag regular expression."""
+        """Check max tags and check against both tag regular expression and required tag list."""
         if value is not None and len(value) > constants.MAX_TAGS_COUNT:
             self.value_error(
                 f"Expecting no more than {constants.MAX_TAGS_COUNT} tags "
                 "in metadata")
         for tag in value:
             if not re.match(constants.NAME_REGEXP, tag):
                 self.value_error(f"'tag' has invalid format: {tag}")
+        if not any(tag in REQUIRED_TAG_LIST for tag in value):
```

Comment:
This check will probably need to be configurable, defaulted to off. We want this check to occur in Community Galaxy and Automation Hub, but this would break most users of `pulp_ansible` independent of `galaxy` and `galaxy_ng`.